deployability/modules/testing/tests/test_agent/test_basic_info.py

```diff
@@ -0,0 +1,84 @@
+# Copyright (C) 2015, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is a free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+import pytest
+import re
+
+from ..helpers.agent import WazuhAgent, WazuhAPI
+from ..helpers.constants import WAZUH_ROOT
+from ..helpers.generic import HostConfiguration, HostInformation, GeneralComponentActions, Waits
+from ..helpers.logger.logger import logger
```

Comment:
```suggestion
from modules.testing.utils import logger
```

We need to implement logger with the new fixes

Reply:
Done https://github.com/wazuh/wazuh-qa/pull/5225/commits/3220fb711cbfc202b9591d26d10576732b8cd2e8